docs/source/roles/role-edpm_network_config.rst

```diff
@@ -1,10 +1,33 @@
-=============================
+============================
 Role - edpm_network_config
-=============================
+============================
 
-.. note::
-  When the `edpm_network_config_tool` is set to `'os-net-config'`, the `ctlplane_gateway_ip` and `ctlplane_ip`
-  variables must be set on the host.
```

Comment:
is this no  longer true?

Reply:
True - but it will be applicable to both "tools" I guess.
Also, this param was originally part of undercloud.conf - now it is defined as part of network definition. For rhoso, still not sure if user will use same control plane for ocp/edpm nodes. Removed it to avoid confusion.

Reply:
that makes sense, thank you